php/src/Docker/DockerActionManager.php

```diff
@@ -482,7 +486,9 @@ public function ConnectContainerToNetwork(Container $container)
 
     public function StopContainer(Container $container) {
         $url = $this->BuildApiUrl(sprintf('containers/%s/stop?t=%s', urlencode($container->GetIdentifier()), $container->GetMaxShutdownTime()));
-        $this->guzzleClient->post($url);
+        try {
+            $this->guzzleClient->post($url);
+        } catch (\Exception $e) {}
```

Comment:
Maybe have the same logic here to still throw the exception if the status code is not a 404?

Reply:
should be done with https://github.com/nextcloud/all-in-one/pull/96/commits/655eada45c46da7fb2e886afa682f9979bee2861